RecoDataProducts/inc/HelixRecoDir.hh

```diff
@@ -18,6 +18,28 @@ namespace mu2e {
     float slopeSig() const { return std::fabs(_slope/_slopeErr); }
     float chi2ndof() const { return _chi2ndof; }
 
+    // Enum declaration for direction
+    enum PropDir {
+      upstream,
```

Comment:
Please specify at least the initial value. It would be intuitive if you set ambiguous=0, downstream=1, upstream=-1